templates/zerver/help/send-timestamps.md

```diff
@@ -0,0 +1,26 @@
+# Send timestamps
+
+For communities distributed across the world, it is often difficult to
+convey time related information like meetings or events to everyone in
+chat. Zulip has a powerful feature to help: timestamps.
+
+The same timestamp appears differently to different people. For example,
+a person in San Francisco would see:
+
+> Let's have the next meeting on *Thu, May 28 2020, 1:00 AM*.
+
+While someone in India would see:
+
+> Let's have the next meeting on *Thu, May 28 2020, 1:30 PM*.
+
+### Insert a timestamp in your message
+
+{start_tabs}
+
+{!start-composing.md!}
+
+2. Type `!time` and select the option to "pick a date/time".
+
+3. Use the datepicker to select a day and time.
+
+{end_tabs}
```

Comment:
Maybe add a sentence at the end that you can also enter timestamps in other formats, e.g. a UNIX timestamp?